house/views.py

```diff
@@ -1,9 +1,14 @@
-from house.constants import HOME_PAGE_ROUTE, GLOBAL_PAGE_ROUTE
+from django.http import HttpResponseRedirect
+from house.constants import HOME_PAGE_ROUTE, HOUSE_PAGE_ROUTE, GLOBAL_PAGE_ROUTE, ADD_EXPENSE_ROUST
+from expenses.models import Expenses
+from django.shortcuts import render, get_object_or_404
 from .models import House
-from django.shortcuts import render
+from .forms import ExpenseForm
 
 
 def home_page(request):
+    # this line is needed till the login of the mine page will create and merge
```

Comment:
So your PRs dependant on the Login Page PR? #80 
If so, please add at the top comment

Reply:
I dont think it depends on the login PR